packages/date-parser/src/parsers/xAgo.js

```diff
@@ -23,7 +24,10 @@ parser.extract = (text, ref, match, opt) => {
   const value = parseInt(match[2]) * (isPast ? -1 : 1);
   const duration = match[5];
 
-  let refDateStruct = dateStructFromDate(ref);
+  const { timezone } = opt;
+  const adjustedRef = adjustRefdate(ref, dateUnit, timezone);
+
+  let refDateStruct = dateStructFromDate(adjustedRef);
```

Comment:
why do we sometimes pass down `timezone` to `dateStructFromDate` and sometimes we don't?